plugins/plugin-docker/che-plugin-openshift-client/src/main/java/org/eclipse/che/plugin/openshift/client/OpenShiftConnector.java

```diff
@@ -1318,9 +1319,15 @@ private ContainerInfo createContainerInfo(Service svc,
         info.setNetworkSettings(networkSettings);
         info.setHostConfig(hostConfig);
         info.setImage(imageInfo.getConfig().getImage());
+        info.setState(getContainerState(pod));
         return info;
     }
 
+    private ContainerState getContainerState(final Pod pod) {
+        ContainerState state = new ContainerState();
+        state.setStatus(pod.getStatus().getPhase());
```

Comment:
Not sure the kubernetes pod phase match with docker status
https://kubernetes.io/docs/concepts/workloads/pods/pod-lifecycle/#pod-phase

> status=(created|restarting|running|removing|paused|exited|dead)
https://docs.docker.com/engine/api/v1.29/#operation/ContainerList